internal/testutils/coverage.go

```diff
@@ -51,33 +51,40 @@ func fqdnToPath(t *testing.T, path string) string {
 	return ""
 }
 
-// CoverDirEnv returns the cover dir env variable to run a go binary.
+// CoverDirEnv returns the cover dir env variable to run a go binary, if coverage is enabled.
 func CoverDirEnv() string {
-	if CoverDir() == "" {
+	if CoverDirForTests() == "" {
 		return ""
 	}
-	return fmt.Sprintf("GOCOVERDIR=%s", CoverDir())
+	return fmt.Sprintf("GOCOVERDIR=%s", CoverDirForTests())
 }
 
-// AppendCovEnv returns the env needed to enable coverage when running a go binary.
+// AppendCovEnv returns the env needed to enable coverage when running a go binary,
+// if coverage is enabled.
 func AppendCovEnv(env []string) []string {
-	envVar := CoverDirEnv()
-	if envVar == "" {
+	coverDir := CoverDirEnv()
+	if coverDir == "" {
 		return env
 	}
-	return append(env, envVar)
+	return append(env, coverDir)
 }
 
-// CoverDir parses the arguments to find the cover profile file.
-func CoverDir() string {
+// CoverDirForTests parses the test arguments and return the cover profile directory,
```

Comment:
Isn't a bit a repetition adding `Tests` suffix when this is part of a `testutils` package? :thinking:

Reply:
Right, but the goal is to say we are relying on tests itself (this coverdir only works for `go test` and not for coverage for binary package). Hence this name!